### PR TITLE
Working on rendering team name

### PR DESCRIPTION
**Pull request by FredAmstutz**

### CODE REVIEW

File: views/team.handlebars [hunk 1]
```diff
@@ -2,6 +2,8 @@
     <div class="team-name">
         <h1>Name your team:</h1>
         <input type="text">
+        <button>Set Team Name</button>
+            <h1></h1>
```
Comment (WilliamfShaw) [inline views/team.handlebars:6]: ?????

File: public/styles/style.css [hunk 2]
```diff
@@ -201,9 +196,28 @@ body:before {
...
+}
+
 .team-name input {
     width: 200px;
 }
+
+#team-name-heading {
+    height: 60px;
+    width: 100%;
+    text-align: center;
```
Comment (WilliamfShaw) [inline public/styles/style.css:212]: Think about combining text-aligns to reduce duplication. 

```
body, 
#team-name-heading {
     text-align: center;
}
```

Comment (WilliamfShaw) [reply public/styles/style.css:212]: Or, better even, add a class that you can ad in markup that will allow you to text align things, ex:

```
.center-text {
    text-align: center;
}
```

`<h1 class="center-text"></h1>`

Thoughts?

File: views/team.handlebars (hunk 1 above)
Comment (FredAmstutz) [reply views/team.handlebars:6]: fixed this.

File: public/styles/style.css (hunk 2 above)
Comment (FredAmstutz) [reply public/styles/style.css:212]: Working on it!

**Pull request closed without merging**